src/org/opensolaris/opengrok/search/Summary.java

```diff
@@ -75,7 +75,7 @@ protected static String htmlize(String q) {
 
     private static final Fragment[] FRAGMENT_PROTO = new Fragment[0];
 
-    /** Adds a fragment to a summary.*/
+    /* Adds a fragment to a summary.*/
```

Comment:
Why remove javadoc formatting? I think in this place it makes sense.

Reply:
javadoc complains about missing @param and the comment is self-explaining.